lib/src/configuration.dart

```diff
@@ -72,7 +76,8 @@ class Configuration {
     var parser = ArgParser()
       ..addOption('password', abbr: 'p')
       ..addOption('certificate', abbr: 'c')
-      ..addOption('version', abbr: 'v');
+      ..addOption('version', abbr: 'v')
+      ..addFlag('debug');
```

Comment:
```suggestion
      ..addFlag('debug', abbr: 'd');
```
should we add short flag?